src/tx-format/transaction.md

```diff
@@ -123,9 +124,9 @@ Given helper `sum_variants()` that sums all variants of an enum.
 Transaction is invalid if:
 
 - Any input is of type `InputType.Contract` or `InputType.Message` where `input.dataLength > 0`
+- Any input uses non-base asset.
 - Any output is of type `OutputType.Contract` or `OutputType.Variable` or `OutputType.Message`
-- More than one output is of type `OutputType.Change` with `asset_id` of zero
-- Any output is of type `OutputType.Change` with non-zero `asset_id`
+- Any output is of type `OutputType.Change` with non-base `asset_id`
```

Comment:
nit: Out-of-scope for this PR. Still nice fixes and having them here is ok for me.